crates/sozo/ops/src/migration/mod.rs

```diff
@@ -44,6 +44,9 @@ pub struct ContractMigrationOutput {
     pub base_class_hash: Felt,
 }
 
+// TODO: read deployment manifest to find diff in `writes` for auto authorize
```

Comment:
Do we need to read deployment manifest if we are fetching events from the chain?

Reply:
nope, was stale comment, updated.